### PR TITLE
Jakarta Servlet 6.0 Plan Review

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record: https://projects.eclipse.org/projects/ee4j.servlet/releases/6.0.0

### CODE REVIEW

**Comment by marceloancelmo** [comment]
LGTM
I will start the ballot process soon

**Review by marceloancelmo — APPROVED** [review]

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:9]: This line should be removed (or replaced with a reference to the Jakarta EE 10 release plan, but that doesn't exist yet...)  :-)

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:10]: I would temporarily blank out these references until you have real documents to point at.

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:11]: Same here...

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:13]: And here...

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:15]: And here...

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:28]: Thank you for explicitly explaining the reasons behind the major version update.

**Review by kwsutter — COMMENTED** [review]
Just a few items that I would clean up before merging...

File: servlet/6.0/_index.md
Comment (kwsutter) [inline servlet/6.0/_index.md:8]: This link looks correct, but the actual release record has a version of 6.0.0 instead of 6.0.  Specifications should only have a two digit version (major.minor).  So, like I mentioned, the link looks correct, but the actual release record should be updated to be 6.0 instead of 6.0.0.

**Review by kwsutter — COMMENTED** [review]
Found another item...

File: servlet/6.0/_index.md
Comment (edbratt) [inline servlet/6.0/_index.md:3]: This date controls when the page is rendered, once this PR is pushed. Please set it to the merge date, or earlier. Otherwise, this page will not appear until Oct 15 which is not what we want.

**Review by edbratt — APPROVED** [review]
Please update the meta-data date field. Otherwise LGTM.

File: servlet/6.0/_index.md
Comment (edbratt) [reply servlet/6.0/_index.md:10]: You could simply remove the URL target and leave these as plain text w/o any link

**Review by karianna — APPROVED** [review]
LGTM pending the last few fixes from other reviewers

**Pull request merged**
